### PR TITLE
Feature/us17534 contact page

### PR DESCRIPTION
**Pull request by jakecatron**

### CODE REVIEW

**Comment by jakecatron** [comment]
This one is also updated and ready for review, thanks!
@seancdavis

File: _includes/_contact-form.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<section class="soft push-ends">
+    <div class="container">
```
Comment (silasbaur) [inline _includes/_contact-form.html]: I would put `soft`, `push-ends`,  and the `container` class all together if I was doing this

File: _assets/stylesheets/components/_jumbotron.scss [hunk 2]
```diff
@@ -6,6 +6,13 @@
   margin-bottom: 0px;
   position: relative;
 
+
+  .text-bottom-left {
```
Comment (silasbaur) [inline _assets/stylesheets/components/_jumbotron.scss]: https://github.com/crdschurch/crds-music/pull/18#discussion_r308248821

Comment (silasbaur) [reply _assets/stylesheets/components/_jumbotron.scss]: I fixed this and it is in development so you can pull that from development into your branch

**Pull request merged**